src/lib/commandBarConfigs/projectsCommandConfig.ts

```diff
@@ -41,12 +41,11 @@ export const projectsCommandBarConfig: StateMachineCommandSetConfig<
       name: {
         inputType: 'options',
         required: true,
-        options: [],
-        optionsFromContext: (context) =>
-          context.projects.map((p) => ({
+        options: (_, context) =>
+          context?.projects.map((p) => ({
             name: p.name!,
             value: p.name!,
```

Comment:
Can you remove these non-null assertions? They're not needed.

```suggestion
            name: p.name,
            value: p.name,
```

Reply:
Just to clarify, the non-null assertions are the same as the `as` keyword? it's like "trust be bro" to typescript? and that's why we should not use them?

Reply:
Correct. It's just like using `as`, except the only change to the type is that it's not null and not undefined.

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator